pom.xml

```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>com.mainstreethub</groupId>
     <artifactId>parent-pom</artifactId>
-    <version>1.5.1</version>
+    <version>1.6.0</version>
   </parent>
 
   <artifactId>dropwizard-api-key-bundle</artifactId>
-  <version>1.0.1-SNAPSHOT</version>  <!-- Make sure to keep this in sync with the dropwizard version below. -->
+  <version>1.1.4-SNAPSHOT</version>  <!-- Make sure to keep this in sync with the dropwizard version below. -->
```

Comment:
We will likely want to set this up to have our new versioning scheme, where we have a build release number:
`1.1.4-1-SNAPSHOT`